sbin/build.sh

```diff
@@ -20,7 +20,13 @@ WORKING_DIR=$1
 TARGET_DIR=$2
 OPENJDK_REPO_NAME=$3
 BUILD_FULL_NAME=$4
-JVM_VARIANT=${5:-normal}
+JVM_VARIANT=${5:=normal}
+NOBUILD=$6
```

Comment:
Instead of saying NOBUILD we could say RUN_TESTS_ONLY...something like that. but lets work with this first and a new PR can help change the language of the idea.